go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.16.0
 	golang.org/x/net v0.8.0
 	google.golang.org/grpc v1.55.0
+	gopkg.in/yaml.v3 v3.0.1
 	sigs.k8s.io/yaml v1.3.0
```

Comment:
Seems like sigs.k8s.io/yaml is only used in service.go which I _believe_ can be drop-in replaced with gopkg.in